src/components/Tables/Table.cy.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import Table from "./Table";
+
+describe("<Table />", () => {
+    const data = [
+        {
+            full_name: "Tom",
+            phone_number: 123456,
+        },
+        {
+            full_name: "Sam",
+            phone_number: 999,
+        },
+        {
+            full_name: "Harper Garrett",
```

Comment:
where did these names come from? I love it 😆

Reply:
https://json-generator.com/ lol